samples/msal-angular-v2-samples/angular10-sample-app/src/app/profile/profile.component.ts

```diff
@@ -26,7 +23,7 @@ export class ProfileComponent implements OnInit {
   }
 
   getProfile() {
-    this.http.get(GRAPH_ENDPOINT)
+    this.http.get('https://graph.microsoft.com/v1.0/me')
```

Comment:
I'd keep the constant.  Curious: Should all of our samples being using the official MS Graph sdk to make requests to MS Graph: https://github.com/microsoftgraph/msgraph-sdk-javascript

Reply:
We use Graph SDK whenever possible in actual sample repos ([e.g.](https://github.com/Azure-Samples/ms-identity-javascript-angular-tutorial/tree/main/2-Authorization-I/1-call-graph)). Since these are samples for testing primarily, I'm inclined to keep these lean, but we can certainly work on this in another PR if decided cc @peterzenz

Reply:
coming back to the constant, it doesn't really have a use here -ideally we would centralize these and import from one location, but due to the angular module load ordering it would run into circular dependency unless we add a separate config file. I'm just shy of doing this in this PR due to its size.

Reply:
This seems OK to me then.  Thanks.